easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2018.01.01-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PackedBinary"
+
+name = "EvidentialGene"
+version = '2018.01.01'
+
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://arthropods.eugenes.org/EvidentialGene/'
+description = """EvidentialGene is a genome informatics project for 
+ "Evidence Directed Gene Construction for Eukaryotes",
+ for constructing high quality, accurate gene sets for
+ animals and plants (any eukaryotes), being developed by
+ Don Gilbert at Indiana University, gilbertd at indiana edu."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['ftp://arthropods.eugenes.org/evigene_old/']
```

Comment:
URL is wrong? Also, prefer `http` over `ftp` when possible.

`http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/`

Reply:
url is not wrong, for me it works. but http used.